packages/core/src/UserSettings.ts

```diff
@@ -73,19 +73,15 @@ export async function prepareUserSettings(): Promise<IUserSettings> {
  * @returns
  */
 
-export async function getEncryptionKey(): Promise<string | undefined> {
+export async function getEncryptionKey(): Promise<string> {
 	if (process.env[ENCRYPTION_KEY_ENV_OVERWRITE] !== undefined) {
-		return process.env[ENCRYPTION_KEY_ENV_OVERWRITE];
+		return process.env[ENCRYPTION_KEY_ENV_OVERWRITE] as string;
 	}
 
 	const userSettings = await getUserSettings();
 
-	if (userSettings === undefined) {
-		return undefined;
-	}
-
-	if (userSettings.encryptionKey === undefined) {
-		return undefined;
+	if (userSettings === undefined || userSettings.encryptionKey === undefined) {
+		throw new Error('Encryption key not set');
```

Comment:
could we throw `RESPONSE_ERROR_MESSAGES.NO_ENCRYPTION_KEY` here? because we are always catching this error and overwrite it. This would simplify things I think. wdyt?

Reply:
Unfortunately no, this file resides in the `core` package and the constant is in `cli`. What I could do is use the same string, but unfortunately I cannot relate to it here.